### ISSUE TITLE
[FR] Go to the note containing the image

### ISSUE THREAD

**Issue by javiavid** [TomNCatz/obsidian-gallery#9]
Sometimes you see an image and you would like to go to the note where it is contained.
It would be great if the note opened by pressing CTRL + click on the image.

**Comment by TomNCatz** [comment]
There can be multiple notes where it is contained, do you mean ctrl click to take you to the meta file? or perhaps open the image itself?

**Comment by javiavid** [comment]
It's true, I hadn't thought about it.  In my vault I don't have any images
repeated in several notes.  Perhaps in such a case, a warning or a selector
could appear?  I think it's very useful to be able to go to the note,
sometimes you don't remember the name of the note but you know it contains
an image or when you see the image it reminds you of an old note that you
want to review.

**Comment by TomNCatz** [comment]
Currently the image meta file contains a list of backlinks to anywhere the image is referenced. Maybe I can find a way to add a right click option to open that to the side when you click an image. I can see other use cases for such a feature anyway.

**Comment by javiavid** [comment]
Maybe I don't understand it well, but in the metadata I don't see the possibility of going to the TEST3 note that contains the image.

<img width="1267" alt="image" src="https://github.com/TomNCatz/obsidian-gallery/assets/7627504/2ab668dd-1819-401c-8cc7-89fc354565a2">

**Comment by TomNCatz** [comment]
Firstly, as I went to demonstrate this I found a bug where the backlinks don't work in the side panel for some reason, so I'm going to make a separate bug ticket for that.
Secondly, once I FIX the side panel back links, they are marked on this image in pink. Also marked in green is a link to the actual image file. If you open the meta file the links currently work, but I will figure out how to get them working again in the side panel too.
![backlinks](https://github.com/TomNCatz/obsidian-gallery/assets/33710598/9d04c479-d2d3-40de-8399-2efd8d89ffc6)
